src/interfaces/IReservoirPriceOracle.sol

```diff
@@ -53,6 +54,7 @@ interface IReservoirPriceOracle {
      */
     function getLargestSafeQueryWindow() external view returns (uint256);
 
+    // REVIEW: Who calls this?
```

Comment:
advanced / custom users of the oracle who want to interact with the underlying raw oracle data. The vault will be using the consumer version `getQuotes`

Reply:
So there's no planned users of this, should we not delete it then to reduce audit scope?

Reply:
okay rm-ed